### DISCUSSION TITLE
findMany returns zero entries

### DISCUSSION THREAD

**Discussion by i4lina** [prisma/prisma#11888] [Q&A] [answered]
Hi, I came across something strange. I have a database with a table called postBackup. If I look up the table using prisma studio I'm able to browse all the entries without problems. However, if I run ` await prisma.postBackup.findMany()` I always get 0 results. I have run all the migrations and I generated the prisma client on the latest schema. How can I solve this?

I'm running the latest prisma version. Thanks for your help!

**Comment by gustavo-maurina** [comment, marked as answer]
Is the value returned from this piece code an actual empty array? Or is it something else? If you put that inside a try-catch block and throw the Error, is there nothing being thrown?

**Reply by i4lina** [reply]
I was trying with this script 
```
const main = async () => {
  const postsToChange = await prisma.postBackup.findMany();
  console.log(postsToChange);
};

main();
```
And all I got was `0`, as per the number. I tried again and now it seems to work. I don't know what was happening, I tried running the code several times, changing the query parameter but the results where always the same.

**Reply by gustavo-maurina** [reply]
Good that its working, just try wrap a try-catch block around that and throw errors, you might find useful errors that way

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
